code/modules/mob/living/basic/space_fauna/spider/young_spider/young_spider_subtypes.dm

```diff
@@ -116,30 +116,30 @@
 	poison_per_bite = 2
 	poison_type = /datum/reagent/toxin/acid
 
-/mob/living/basic/young_spider/tangle/Initialize(mapload)
+/mob/living/basic/spider/growing/young/tangle/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/healing_touch,\
 		heal_brute = 10,\
 		heal_burn = 10,\
 		heal_time = 3 SECONDS,\
 		self_targetting = HEALING_TOUCH_SELF_ONLY,\
 		interaction_key = DOAFTER_SOURCE_SPIDER,\
-		valid_targets_typecache = typecacheof(list(/mob/living/basic/giant_spider/tangle)),\
+		valid_targets_typecache = typecacheof(list(/mob/living/basic/spider/giant/tangle)),\
```

Comment:
Can you change this to /mob/living/basic/spider/growing/young/tangle? Then I can close my fix pr